src/js/structs/Service.js

```diff
@@ -249,6 +264,20 @@ module.exports = class Service extends Item {
 
   }
 
+  handleRename(fromName, toName) {
```

Comment:
Do we actually want to be able to modify a service in place?
This is usually a one-way creation from a json of some sort